### PR TITLE
@dzucconi - sets up context for iconic works

### PR DESCRIPTION
**Pull request by broskoski**

### CODE REVIEW

File: lib/artist_blacklist.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export default [
+  '52016252275b2453200000ca', // alec-monopoly
+  '4e975df46ba7120001001fe2', // mr-brainwash
+  '502c4791457f0100020001c2', // retna
+  '4d8b92904eb68a1b2c00022e', // invader
+  '4ddd81b9ff91ce00010012c8', // shepard-fairey
+  '5098106a7f47bb0002000e4f', // yoshitomo-nara
+  '4e934002e340fa0001005336', // kaws
+  '4deaa8fa7ae5cf0001008eb3', // victor-vasarely
+];
```
Comment (mzikherman) [inline lib/artist_blacklist.js]: clint-eastwood-06

File: schema/home/fetch.js [hunk 2]
```diff
@@ -26,3 +28,15 @@ export const featuredGene = (accessToken) => {
...
 };
+
+export const iconicArtists = () => {
+  return delta('/', {
+    method: 'fetch',
+    n: 9,
+    name: 'artist_search_2t',
+  }).then((trending) => {
+    const clonedTrending = clone(trending);
+    forEach(blacklist, (id) => delete clonedTrending[id]);
```
Comment (dzucconi) [inline schema/home/fetch.js]: https://lodash.com/docs#omit ?

Comment (broskoski) [inline schema/home/fetch.js]: ah yes thank ye

**Comment by dzucconi** [comment]
I approve.

**Pull request merged**